.gitignore

```diff
@@ -32,6 +32,7 @@ bld/
 
 # Logs
 Logs/
+log* 
```

Comment:
This should ignore all files that has log at the start of name? (never really paid attention to how .gitignore works :D )

Reply:
As I understand, yes :D And it seems to be working as intended